apidef/oas/authentication.go

```diff
@@ -417,15 +437,15 @@ func (s *Scopes) Fill(api apidef.APIDefinition) {
 	}
 }
 
-func (s *Scopes) ExtractTo(api *apidef.APIDefinition) {
-	api.JWTScopeClaimName = s.ClaimName
+func (s *Scopes) ExtractTo(scopeClaim *apidef.ScopeClaim) {
+	scopeClaim.ScopeClaimName = s.ClaimName
 
 	for _, v := range s.ScopeToPolicyMapping {
```

Comment:
@furkansenharputlu  @jeffy-mathew  can this be transformed in more like a generic util func? ArrToMap or so?